frontend/src/assets/Dashboard/Table.tsx

```diff
@@ -1,18 +1,7 @@
-				fill="#1668DC"
-			/>
-		</svg>
-	);
+import { Table } from 'lucide-react';
+
+function TableIcon(): JSX.Element {
+	return <Table size={30} />;
```

Comment:
Why not consume the `Table` icon directly in the component ?

Reply:
Used SVG instead to match other icons.